contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -300,6 +355,64 @@ def _create_movieproc(time_range, kind="blank"):
     return movieproc
 
 
+def _add_source_bounds(media_ref, src, context):
+    if not hasattr(media_ref, "available_image_bounds"):
+        return
```

Comment:
Maybe this is my c++ programmer paranoia, but I was more thinking if the RV adapter was out of sync with the otio version.  For example, if I have a otio version installed from the `main` branch, and I build the package from my `spatial coordinates` branch, the `main` branch otio would not have the `available_image_bounds` property.

Reply:
Resolving this seems like a pre-requisite for merging to main. How about we go with the suggestion? Maybe maintain the hasattr variant as a local-dev-only patch for convenience, rather than committing it to the repo?